### DISCUSSION TITLE
OpenChat 3.5 correct prompt format

### DISCUSSION THREAD

**Discussion by arch-btw** [ggml-org/llama.cpp#3924] [General]
Hi,

I was just wondering what the correct prompt format is for OpenChat with llama.cpp

So far I have this:

    main \
    -m openchat_3.5.Q5_K_M.gguf \
    --in-prefix "GPT4 Correct User: " \
    --in-suffix "<|end_of_turn|>GPT4 Correct Assistant:" \
    -p 'You are a helpful assistant.<|end_of_turn|>' \
    -e -n -1 -t 8 -i --color --interactive-first --log-disable

Partly due to this comment from the repo owner:

https://huggingface.co/openchat/openchat_3.5/discussions/5#65448109b4a3f3a2f486fd9d

Is this correct? Thank you!

**Comment by TikaToka** [comment]
have you solved?

**Comment by arch-btw** [comment]
@TikaToka not yet. It works but I don't know if it's optimal. Sorry.

**Reply by TikaToka** [reply]
@arch-btw Thanks I am also trying it. I will share if I find some solution
